core/README.md

```diff
@@ -61,13 +61,31 @@ Baseline components are required for all other automation areas.
 
 ## Container components
 
-1. Install the container build layer to your target. In this case, you
-   install the container build for the reference distribution named **Poky**.
+1. Install the container build layer to your target.  The script
+   naming convention is
+   `setup_ci_container_&lt;distro&rt;[.&lt;release&rt;].sh`.
+
+   In the Poky case, you install the container build using the script
+   with the name `poky` in it.
 
    ```bash
    ./setup_ci_container_poky.sh $PREFIX $SECRET_ARN
    ```
 
+    In the TI (Arago) case, you will need to be more specific.
+
+   ```bash
+   ./setup_ci_container_ti.dunfell.sh $PREFIX $SECRET_ARN
+   ```
+
+   If you have forked the meta-aws-ci repository and need to use the
+   repo from your own context, set the `GITHUB_REPO` variable. For
+   example:
+
+
+   ```bash
+   GITHUB_ORG=rpcme ./setup_ci_container_ti.dunfell.sh $PREFIX $SECRET_ARN
```

Comment:
Why not as a param?

Reply:
because it's optional - then could have another required which messes things up.  I thought today it's probably time to use getopt but ... that's a bigger lift.